sklearn/metrics/tests/test_ranking.py

```diff
@@ -536,10 +536,10 @@ def test_precision_recall_curve_toydata():
         assert_array_almost_equal(r, [1, 0.])
         assert_almost_equal(auc_prc, .75)
 
-        y_true = [0, 0]
-        y_score = [0.25, 0.75]
-        assert_raises(Exception, precision_recall_curve, y_true, y_score)
-        assert_raises(Exception, average_precision_score, y_true, y_score)
+        # y_true = [0, 0]
+        # y_score = [0.25, 0.75]
+        # assert_raises(Exception, precision_recall_curve, y_true, y_score)
```

Comment:
This is the behaviour change, it uses to warn because of a division by zero which was turned into an error by the `np.seterr('raise')` context manager. Remove the `np.seterr('raise')` and make sure that you test the result of `precision_recall_curve`.

You don't need to add more tests for the edge cases since they were already tested, you just need to change what we expect from the results.